sdk/keyvault/azure-security-keyvault-keys/test/ut/test_consts.hpp

```diff
@@ -0,0 +1,152 @@
+namespace Azure { namespace Security { namespace KeyVault { namespace Keys { namespace Test {
```

Comment:
The region where the test runs is configured in the ci.yml file. The "Location:" property specifies the location in which the test resources are created.

Reply:
done